### ISSUE TITLE
Schema problem... (for free)

### ISSUE THREAD

**Issue by Mago7891** [raventools/schema-creator#145]
I got a problem with schema on a website.

**Comment by Mago7891** [comment]
Usually, when I check the schema, I use Google on this link https://developers.google.com/structured-data

But I got a product for free and the system says me "Error" if I let this field with "For free" written.

<p itemprop="price" class="price"><span class="amount">0</span></p>

The system prefers the number 0

How can I solve this trouble without writing 0 on the website ?

http://www.foreverbeplus.com/prodotto/abc-del-cuore/

This is the webpage I was talking about

**Comment by stephenyeargin** [comment]
Flagging this as an enhancement. Supposing that it isn't letting you leave the price attribute empty?

**Comment by Mago7891** [comment]
Yeah. I don't know how to edit the price attribute, without showing 0 on my website.
